modules/apps/commerce/commerce-payment-service/src/main/java/com/liferay/commerce/payment/internal/configuration/FunctionCommercePaymentIntegrationConfiguration.java

```diff
@@ -18,32 +18,37 @@
 )
 @Meta.OCD(
 	factory = true,
-	id = "com.liferay.commerce.payment.internal.configuration.FunctionCommercePaymentIntegrationConfiguration"
+	id = "com.liferay.commerce.payment.internal.configuration.FunctionCommercePaymentIntegrationConfiguration",
+	localization = "content/Language",
+	name = "function-commerce-paymentIntegration-configuration-name"
 )
 public interface FunctionCommercePaymentIntegrationConfiguration {
 
-	@Meta.AD(type = Meta.Type.String)
+	@Meta.AD(name = "authorizePath", required = false, type = Meta.Type.String)
 	public String authorizePath();
 
-	@Meta.AD(type = Meta.Type.String)
+	@Meta.AD(name = "cancelPath", required = false, type = Meta.Type.String)
 	public String cancelPath();
 
```

Comment:
@AkramYoussoufi, if there isn't any reason, please don't change the previous logic about `required `.